cmd/install/addon/cmd.go

```diff
@@ -132,15 +136,71 @@ func run(_ *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
-	reporter.Warnf("Once installed, add-ons cannot be uninstalled")
-	if confirm.Confirm("install add-on '%s' on cluster '%s'", addOnID, clusterKey) {
-		reporter.Debugf("Installing add-on '%s' on cluster '%s'", addOnID, clusterKey)
-		err = clusterprovider.InstallAddOn(clustersCollection, clusterKey, awsCreator.ARN, addOnID)
-		if err != nil {
-			reporter.Errorf("Failed to add add-on installation '%s' for cluster '%s': %s", addOnID, clusterKey, err)
-			os.Exit(1)
-		}
-		reporter.Infof("Add-on '%s' is now installing. To check the status run 'rosa list addons -c %s'",
-			addOnID, clusterKey)
+	if !confirm.Confirm("install add-on '%s' on cluster '%s'", addOnID, clusterKey) {
+		os.Exit(0)
+	}
+
+	parameters, err := clusterprovider.GetAddOnParameters(ocmClient.Addons(), addOnID)
+	if err != nil {
+		reporter.Errorf("Failed to get add-on '%s' parameters: %s", addOnID, err)
```

Comment:
I don't know whether it actually makes a difference, since a lot of these errors are bubbled up from arbitrary API messages, but just in case it's better to be consistent :+1: